### PR TITLE
Adds native spelunker support

### PR DESCRIPTION
**Pull request by dxenes1**
- [x] create example tasks
- [x] test queue functionality
- [x] test submission data functionality

### CODE REVIEW

**Comment by hannah-martinez** [comment]
The queue, workspace, workspace buttons, and dashboard all seem work correctly from a UI perspective. The `Inspect Task` tab doesn't seem to work locally. Not sure if that would be fixed through deployment.

**Comment by hannah-martinez** [comment]
I noticed some weird CSS going on. Some of the text boxes are squished. 
image

I was able to fix this with some more CSS classes in `workspace.css`:
```
.neuroglancer-display-dimensions-widget-name,
.neuroglancer-position-dimension-scale,
.neuroglancer-position-dimension-coordinate,
.neuroglancer-position-dimension-name{
  box-sizing: content-box;
}
```
These were the classes I noticed that needed the change. There might be more.

File: neuvue_project/templates/about.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h2 class="pt-5 pb-3"> Collaborators </h2>
         <h2 class="pt-5 pb-3"> Contact Us </h2>
 
         <p>
-            We'd love to hear from you! Reach out at <a class="text-secondary-color-activated" href="mailto:info@neuvue.io">info@neuvue.io</a> with questions or if you are interested in collaborating with us.
+            We'd love to hear from you! Reach out at <a class="text-secondary-color-activated" href="mailto:info@neuvue.io">info@bossdb.org</a> with questions or if you are interested in collaborating with us.
```
Comment (hannah-martinez) [inline neuvue_project/templates/about.html:85]: Very glad you are changing this. Can you change the `mailto:` link as well?

File: neuvue_project/workspace/analytics.py [hunk 2]
```diff
@@ -111,8 +111,10 @@ def create_stats_table(pending_tasks, closed_tasks):
         for namespace, namespace_df in df.groupby("namespace"):
             n_tasks = len(namespace_df)
             m = namespace_df[status].min()
-            average_event_time = (m + (namespace_df[status] - m)).mean().to_pydatetime()
-            changelog.append((average_event_time, n_tasks, status, namespace))
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                average_event_time = (m + (namespace_df[status] - m)).mean().to_pydatetime()
+                changelog.append((average_event_time, n_tasks, status, namespace))
```
Comment (hannah-martinez) [inline neuvue_project/workspace/analytics.py:117]: Why are we ignoring warnings here?

Comment (dxenes1) [reply neuvue_project/workspace/analytics.py:117]: a previous team member hannah gooden reported annoying warnings here
https://github.com/aplbrain/neuvue-app/issues/382

File: neuvue_project/templates/about.html (hunk 1 above)
Comment (dxenes1) [reply neuvue_project/templates/about.html:85]: consider it done!

**Pull request merged**